service/start.go

```diff
@@ -119,6 +119,82 @@ func (svc *service) startNostr(ctx context.Context, encryptionKey string) error
 	return nil
 }
 
+func (svc *service) SubscribeToAppRequests(ctx context.Context, appWalletPubKey string) error {
```

Comment:
This seems duplicated from the existing code. Can we update the existing method to use this one for the original wallet service key?

Reply:
yep, in progress major refactor :)